src/Compilers/CSharp/Test/Emit/Emit/EndToEndTests.cs

```diff
@@ -33,7 +33,7 @@ public void OverflowOnFluentCall()
                     numberFluentCalls = 460;
                     break;
                 case 32 when !isDebug:
-                    numberFluentCalls = 1000;
+                    numberFluentCalls = 1300;
```

Comment:
>1300 [](start = 40, length = 4)

nit: I think it will be possible to restore this to 1350 (and bump all other values by 50 as well). That would close [https://github.com/dotnet/roslyn/issues/25606](https://github.com/dotnet/roslyn/issues/25606)

Reply:
Locally at least I can do that. Will see if it holds up in CI.

Reply:
You could restore the other values as well (+50) to close the linked issue. Thanks

---
In reply to: [263190688](https://github.com/dotnet/roslyn/pull/33907#discussion_r263190688) [](ancestors = 263190688)

Reply:
Thanks!
Add "Closes https://github.com/dotnet/roslyn/issues/25606"?

Reply:
It's in the commit where the change was done. I put it there in case I had to revert that part of the change to get the PR through.